docs/development/packages/update-helm-charts.md

```diff
@@ -61,3 +71,24 @@ Note in some helm charts, fields above in `yaml` files are not hardcoded values
 
 For packages that include CRDs as well as custom resources, the CRDs must be deployed before the rest of the resources. CRDs can't be included in the `templates` directory because the result is a single yaml file applied once. To overcome this issue, CRDs must be defined in their own package under the `templates` directory. Once a CRDs package is ready, a dependency to that package can be declared in the package bundle definition. To add the dependency to the resulting bundle file, add `PACKAGE_DEPENDENCIES=X` to your package `Makefile` in build tooling. If both the CRDs and the actual chart are built from the same project, you'll have to use the same workaround as used in metallb which involves redefining the helm/build and helm/push targets.
 
+
+### Testing
+For testing locally run steps below
+* Run `export IMAGE_REPO='public.ecr.aws/<registry_alias>'`
+* Run `make images` to build any images associated with  package
+* Run `make helm/build` to build the helm chart
+* Run `make helm/push` to push built chart to registry defined in `IMAGE_REPO` above
+
+Once the package/chart is in your personal registry test package
+
+Using helm
+
+```
+helm install {NAME} oci://{chart_registry}/{chart_repo} —version {chart_version} --set sourceRegistry={chart_registry}
+```
+Using EKSA CLI
+
+1. Generate local bundle using [generatebundlefile](https://github.com/aws/eks-anywhere-packages/tree/main/generatebundlefile) utility
```

Comment:
Do we have specific instructions to run here?

Reply:
Yes for those steps are provided in [README](https://github.com/aws/eks-anywhere-packages/tree/main/generatebundlefile#readme)